enterprise/views.py

```diff
@@ -486,6 +489,15 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_uuid, course
             logger.error('Unable to find enterprise customer for UUID: %s', enterprise_uuid)
             raise Http404
 
+        if EnterpriseCourseEnrollment.objects.filter(
+                enterprise_customer_user__enterprise_customer=enterprise_customer,
+                enterprise_customer_user__user_id=request.user.id,
+                course_id=course_id
+        ).exists():
```

Comment:
@douglashall Yup, I've added this functionality in and this should be good to go. It looks like there's a build delay issue on travis right now though, so I'm just waiting for the tests to go through now.